etip/trackers/management/commands/compare_with_exodus.py

```diff
@@ -0,0 +1,99 @@
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from trackers.models import Tracker
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
+EXODUS_API_BASE_PATH = '/api/trackers'
+FIELDS_TO_LOOKUP = [
+    'name',
+    'code_signature',
+    'network_signature',
+    'website'
+]
+LOOKUP_IS_EXACT = 'EXACT_MATCH'
+LOOKUP_IS_DIFFERENT = 'DIFFERENT'
+LOOKUP_MULTIPLE = 'MULTIPLE_IN_ETIP'
+LOOKUP_NOT_FOUND = 'NOT_FOUND_IN_ETIP'
+
+
+class Command(BaseCommand):
+    help = 'Compare trackers stored in Exodus with trackers from ETIP DB'
+
+    def handle(self, *args, **options):
+        exodus_trackers = self.get_all_from_exodus()
+        self.stdout.write(
+            'Retrieved {} trackers from Exodus'.format(len(exodus_trackers))
+        )
+        self.count_from_etip()
+        self.lookup_trackers(exodus_trackers)
+
+    def __exact_lookup(self, tracker_details):
+        for idx, field in enumerate(FIELDS_TO_LOOKUP):
+            kwargs = {
+                'is_in_exodus': True,
+                field + '__iexact': tracker_details.get(field)}
+            results = Tracker.objects.filter(**kwargs)
+            if not results and idx + 1 == len(FIELDS_TO_LOOKUP):
+                return (LOOKUP_NOT_FOUND, None)
+            elif not results:
+                continue
+            elif len(results) != 1:
```

Comment:
results.count() is probably preferable

Reply:
Thanks for the tip, taken it into account (but also refactored the whole function ^^)